pkgs/applications/misc/cura/default.nix

```diff
@@ -3,20 +3,20 @@
 
 mkDerivation rec {
   pname = "cura";
-  version = "4.12.1";
+  version = "4.13";
 
   src = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "Cura";
     rev = version;
-    sha256 = "sha256-QvX9o1nrYmY6zzPcxl+xD6JTMdphzT/is1SMYrISu4o=";
+    sha256 = "sha256-qopY532pNudxaqOiEA5z71TP4Ckp69fZQbK1rghS3JY=";
```

Comment:
This is the sha256 for https://github.com/Ultimaker/Cura/commit/e6830f024161dee17632e178574530c65b9e1072, 4.13 is actually a branch so I can image hash mismatch will still exist.